modules/openapi-generator/src/main/resources/go/model_simple.mustache

```diff
@@ -196,13 +209,13 @@ func (o *{{classname}}) Get{{name}}() {{vendorExtensions.x-go-base-type}} {
 {{#deprecated}}
 // Deprecated
 {{/deprecated}}
-func (o *{{classname}}) Get{{name}}Ok() ({{^isArray}}{{^isFreeFormObject}}*{{/isFreeFormObject}}{{/isArray}}{{vendorExtensions.x-go-base-type}}, bool) {
+func (o *{{classname}}) Get{{name}}Ok() ({{^isArray}}{{^isFreeFormObject}}*{{/isFreeFormObject}}{{/isArray}}{{#isEnum}}{{{classname}}}{{#lambda.pascalcase}}{{{nameInCamelCase}}}{{/lambda.pascalcase}}{{/isEnum}}{{^isEnum}}{{{vendorExtensions.x-go-base-type}}}{{/isEnum}}, bool) {
 	if o == nil{{^isNullable}} || IsNil(o.{{name}}){{/isNullable}}{{#isNullable}}{{#vendorExtensions.x-golang-is-container}} || IsNil(o.{{name}}){{/vendorExtensions.x-golang-is-container}}{{/isNullable}} {
 	{{^isFreeFormObject}}
-		return nil, false
+		return {{#isEnum}}o.{{name}}{{/isEnum}}{{^isEnum}}nil{{/isEnum}}, false
 	{{/isFreeFormObject}}
 	{{#isFreeFormObject}}
-		return {{vendorExtensions.x-go-base-type}}{}, false
+		return {{#isEnum}}{{{classname}}}{{#lambda.pascalcase}}{{{nameInCamelCase}}}{{/lambda.pascalcase}}{{/isEnum}}{{^isEnum}}{{{vendorExtensions.x-go-base-type}}}{{/isEnum}}{}, false
```

Comment:
I probably changed the template in places where it's not necessary (for example, I don't know if `#isFreeFormObject` implies that the var is not an inline enum); it works for my purposes but would need to be cleaned up _if_ we can't instead update the generator to create separate models for inline enums

Reply:
To create inline enum as separate models, there's an option in inline schema resolver to do that 

https://github.com/OpenAPITools/openapi-generator/blob/master/docs/customization.md#inline-schema-naming => RESOLVE_INLINE_ENUMS=true

Reply:
Oh dang!  That's way better.  Thanks!